L1Trigger/L1TGlobal/src/AXOL1TLTemplate.cc

```diff
@@ -0,0 +1,75 @@
+// this class header
+#include "L1Trigger/L1TGlobal/interface/AXOL1TLTemplate.h"
+
+// system include files
+#include <iostream>
+#include <iomanip>
+
+AXOL1TLTemplate::AXOL1TLTemplate() : GlobalCondition() { m_condCategory = l1t::CondAXOL1TL; }
+
+AXOL1TLTemplate::AXOL1TLTemplate(const std::string& cName) : GlobalCondition(cName) {
+  m_condCategory = l1t::CondAXOL1TL;
+}
+
+AXOL1TLTemplate::AXOL1TLTemplate(const std::string& cName, const l1t::GtConditionType& cType)  //not sure we need cType
+    : GlobalCondition(cName, l1t::CondAXOL1TL, cType) {
+  int nObjects = nrObjects();
+
+  if (nObjects > 0) {
+    m_objectType.reserve(nObjects);
+  }
+}
+
+// copy constructor
+AXOL1TLTemplate::AXOL1TLTemplate(const AXOL1TLTemplate& cp) : GlobalCondition(cp.m_condName) { copy(cp); }
```

Comment:
Please move the single line constructor to the header

Reply:
see comment above: we have to decide between making the code more concise and matching the structure of the other types of conditions (CorrelationCondition, MuonShowers, etc)

Reply:
If this structure is default, it is fine to keep it this way to keep consistent with other conditions. At least as far as I am concerned.